addons/website/views/snippets/snippets.xml

```diff
@@ -214,6 +214,11 @@
         </we-select>
     </div>
 
+    <!-- Imagepicker Widget test, do not merge -->
+    <div data-js="ImagepickerTest" data-selector=".s_three_columns">
+        <we-imagepicker data-image="" string="Testing imagepicker Title" data-allow-videos="true"/>
+    </div>
+
     <!-- Background Image -->
     <div data-js="background" string="Background"
         data-selector="section, .parallax, .carousel-item"
```

Comment:
Instead of your ImagepickerTest, you can already adapt the data-js="background" option I guess? Or is it done in one of your other task maybe?

Reply:
Done in a second commit on this same PR :)